content/operate/kubernetes/upgrade/upgrade-redis-cluster.md

```diff
@@ -19,13 +19,43 @@ Redis implements rolling updates for software upgrades in Kubernetes deployments
 
 ## Prerequisites
 
-1. Check [Supported Kubernetes distributions]({{< relref "/operate/kubernetes/reference/supported_k8s_distributions" >}}) to make sure your Kubernetes distribution is supported.
+The following steps ensure you have the minimum versions of all components necessary to upgrade to 7.8.2-2. **Without these minimum versions, the upgrade will freeze and require manual recovery.**
 
-1. Use `kubectl get rec` and verify the `LICENSE STATE` is valid on your REC before you start the upgrade process.
+See the [troubleshooting](#troubleshooting) section for details on recovering a failed upgrade.
 
-1. Verify you are upgrading from Redis Enterprise operator version 6.2.10-45 or later. If you are not, you must upgrade to 6.2.10-45 before upgrading to versions 6.2.18 or later.
+### Kubernetes version
 
-{{<warning>}}**Upgrade cluster operating system** If your databases use modules, you need to update all nodes in the cluster to Redis Enterprise 7.2.4 or later before upgrading your operating system. See [Upgrade a cluster's operating system]({{< relref "/operate/rs/installing-upgrading/upgrading/upgrade-os" >}})in the Redis Enterprise Software documentation for more details.{{</warning>}}
+Check [Supported Kubernetes distributions]({{<relref "/operate/kubernetes/reference/supported_k8s_distributions" >}}) to make sure your Kubernetes distribution is supported by 7.8.2-2. If not, upgrade your Kubernetes distribution before upgrading the Redis operator.
+
+### Redis operator version
+
+Your Redis Enterprise clusters must be running version 7.4.2-2 or later before upgrading to 7.8.2-2. See the [7.4 upgrade]({{<relref "/operate/kubernetes/7.4.6/upgrade">}}) for detailed steps.
+
+### Redis database version
+
+Your Redis databases must be running version 7.2 or later before upgrading your cluster version to 7.8.2-2. See [upgrade databases](#upgrade-databases) for detailed steps. You can find your database version in the [REDB `spec.redisVersion` field]({{<relref "/operate/kubernetes/reference/redis_enterprise_database_api#redisversion" >}}).
+
+### RHEL9-compatible modules
+
+Upgrading to Redis operator version 7.8.2-2 requires migrating your Redis Enterprise nodes to RHEL9 from either Ubuntu 18 or RHEL8. If your databases use modules, you need to manually install modules compatibile with RHEL9.
```

Comment:
the word `requires` may be misleading here. It's not like users are required to take a special action.
Simply upgrading the operator and the Redis Enterprise cluster results in migration from RHEL8/Ubuntu18 to RHEL9.

Reply:
I'll change this to "involves"